crates/extensions/tedge_http_ext/src/messages.rs

```diff
@@ -4,9 +4,6 @@ use http::header::HeaderValue;
 use serde::de::DeserializeOwned;
 use thiserror::Error;
 
-#[derive(Clone, Debug, Default)]
-pub struct HttpConfig {}
-
```

Comment:
I agree to remove this for now, even if this might be useful later. Indeed, the plan was here to set options such as [pool_idle_timeout](https://docs.rs/hyper/latest/hyper/client/struct.Builder.html#method.pool_idle_timeout)